cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -24,15 +32,16 @@ type CredentialsCmd struct {
 type CliContext struct {
 	JWTSourceProvider credentials.JWTSourceProvider
 	STSProvider       credentials.STSProvider
+	Logger            *logrus.Logger
 }
```

Comment:
Since we need to provide a Logger instance to the SPIRE agent connection and not just on application exit, we're promoting the Logger instance to be instantiated up-front and long-lived by sticking it in the CliContext.